cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -69,6 +69,10 @@ type IndexOptions struct {
 
 	// Map from language to scip-ctags, universal-ctags, or neither
 	LanguageMap ctags.LanguageMap
+
+	// The number of CPUs to use for indexing. Defaults to using all available CPUs. If
+	// the server flag -cpu_fraction is set, then this value overrides it.
+	CPUCount int32
```

Comment:
minor: why is this int32? It looks like we use int later.

Reply:
This was more convenient for the protobuf conversion, as there are randomized tests which generate int32 values and check the roundtrip values. I looked into changing the `Server` to be int32 but that created its own inconsistencies!